django/roles/pyenv/tasks/main.yml

```diff
@@ -0,0 +1,41 @@
+---
+- name: Install pyenv dependencies
+  ansible.builtin.apt:
+    pkg:
+     - make
```

Comment:
make sure to filter out the non-relevant packages.

Reply:
They were recommended by pyenv. Just making sure we have all required packages before moving on.